src/driver/amdxdna/amdxdna_drv.c

```diff
@@ -380,16 +400,39 @@ static int amdxdna_pmops_resume(struct device *dev)
 		}
 	}
 
-	XDNA_INFO(xdna, "hardware context resuming...");
+	XDNA_DBG(xdna, "hardware context resuming...");
 	list_for_each_entry(client, &xdna->client_list, node)
 		amdxdna_hwctx_resume(client);
 	mutex_unlock(&xdna->dev_lock);
 
 	return 0;
 }
 
+static int amdxdna_rpmops_suspend(struct device *dev)
+{
+	struct amdxdna_dev *xdna = pci_get_drvdata(to_pci_dev(dev));
+	int ret;
+
+	ret = amdxdna_pmops_suspend(dev);
+
+	XDNA_DBG(xdna, "Runtime suspend done ret: %d", ret);
+	return ret;
+}
+
+static int amdxdna_rpmops_resume(struct device *dev)
+{
+	struct amdxdna_dev *xdna = pci_get_drvdata(to_pci_dev(dev));
+	int ret;
+
+	ret = amdxdna_pmops_resume(dev);
```

Comment:
See amdxdna_rpmops_suspend(), same suggestion. In this case, we can also keep XDNA_INFO() print when device resume from sleep/hibernate for easy debug purpose.